utils/network.go

```diff
@@ -340,3 +345,41 @@ func writeProcSys(path, value string) error {
 	}
 	return err
 }
+
+// Generate a mac address from the passed in string
+func genLocalMacAddr(src string) net.HardwareAddr {
```

Comment:
@tmjd I'm sorry to backtrack here, but I think we should do what @fasaxc suggested here: https://github.com/projectcalico/cni-plugin/issues/340#issuecomment-339349037

Using a fixed mac will reduce a lot of complexity here and should behave just the same, without the scariness of using the container ID to generate the address.

Reply:
Should I track down what is used in OpenStack or just use "EE:EE:EE:EE:EE:EE" like libnetwork uses?